UNRELEASED.md

```diff
@@ -0,0 +1,5 @@
+**Changed:**
+
+- bpk-foundations-common:
+  - Typography related aliases
+  - Some font sizes have been updated to use the correct values
```

Comment:
As some of the existing font sizes have been updated this would be a breaking change as given the size jumps for some of the tokens, their usage in product maybe affected by the bigger sizes

Reply:
These changes will be reverted, so no existing font sizes will change. It wouldn't be breaking then

Reply:
shall we update the description? I guess there's still some breaking changes now with the font weight & letter spacing updates